service/src/test/java/bio/terra/policy/controller/TpsBasicControllerTest.java

```diff
@@ -100,4 +108,27 @@ private void checkAttributeSet(ApiTpsPolicyInputs attributeSet, String expectedR
       }
     }
   }
+
+  /** check that the initial Pao create matches lastUpdated and return lastUpdated */
+  private String checkInitialLastUpdate(UUID paoId) throws Exception {
+    var apiPao = mvcUtils.getPao(paoId);
+    assertEquals(apiPao.getCreatedDate(), apiPao.getLastUpdatedDate());
+    return apiPao.getLastUpdatedDate();
+  }
+
+  /** check that lastUpdated changed and return new lastUpdated */
+  private String checkLastUpdateChanged(String lastUpdated, UUID paoId) throws Exception {
+    var apiPao = mvcUtils.getPao(paoId);
+    assertNotEquals(lastUpdated, apiPao.getLastUpdatedDate());
+    return apiPao.getLastUpdatedDate();
+  }
+
+  private void dumpDates(UUID paoIdA, UUID paoIdB) throws Exception {
```

Comment:
this mostly seems aimed at debugging, remove?
If you're keeping this, can you switch to `Logger` over `System.out.print`?